crates/ruff_linter/resources/test/fixtures/ruff/RUF057.py

```diff
@@ -0,0 +1,59 @@
+inferred_int = 1
+inferred_float = 1.
+
+
+
+round(42)                                         # Error (safe)
+round(42, None)                                   # Error (safe)
+round(42, 2)                                      # Error (safe)
+round(42, inferred_int)                           # Error (safe)
+round(42, 3 + 4)                                  # Error (safe)
+round(42, foo)                                    # No error
```

Comment:
Should we make this an error too if the value is known to be equivalent to an int? The only semantic change that I could see is that `round` no longer throws if `foo` isn't an int. But we could make the fix unsafe to account for this.